assets/scripts/util/fetch_blocking.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * Create blocking fetch requests.
+ * Display a blocking shield when a request is in progress.
+ * Only one of these should be active at any time and
+ * no other fetch / xhr / ajax should exist when they do.
+ */
+import $ from 'jquery'
+import { showBlockingShield, hideBlockingShield, darkenBlockingShield } from '../app/blocking_shield'
+
+let blockingAjaxRequest
+let blockingAjaxRequestDoneFunc
+let blockingAjaxRequestCancelFunc
+
+// This needs to be exported to window for streets/xhr (temp?)
+export let blockingAjaxRequestInProgress = false
```

Comment:
Gotcha.

I am not sure if I can export a getter that's not on an object, so can we momentarily write it as a normal function:

```
export function isBlockingAjaxRequestInProgress () {
    return blockingAjaxRequestInProgress
}
```